src/Blueprint/Blueprint/Models/PSKeyVaultReference.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
@filizt all of these new files should have a license header

Reply:
Done.